modules/distr-tracing-rn-new-features.adoc

```diff
@@ -0,0 +1,40 @@
+////
+Module included in the following assemblies:
+* distr-tracing--release-notes.adoc
+////
+////
+Feature – Describe the new functionality available to the customer. For enhancements, try to describe as specifically as possible where the customer will see changes.
+Reason – If known, include why has the enhancement been implemented (use case, performance, technology, etc.). For example, showcases integration of X with Y, demonstrates Z API feature, includes latest framework bug fixes.
+Result – If changed, describe the current user experience.
+////
+
+[id="distr-tracing-rn-new-features_{context}"]
+== New features and enhancements {ProductName} 2.0.0
+
+This release marks the rebranding of Red Hat OpenShift Jaeger to {ProductName}.  This effort includes the following:
+
+* An updated {ProductShortName} Operator based on Jaeger 1.28.
+
+* Going forward, {ProductName} will only support the `stable` Operator channel. Channels for individual releases are no longer supported.
+
+* A new OpenTelemetry Operator based on OpenTelemetry 0.33.  Note that this Operator is a Technology Preview.
+
+* Adds support for OpenTelemetry protocol (OTLP) to the Query service.
+
+* There is a new distributed tracing icon that appears in the OpenShift OperatorHub.
+
+* Rolling updates to the documentation to support the name change and new features.
+
+* This release also addresses Common Vulnerabilities and Exposures (CVEs) and bug fixes.
+
+== Component versions supported in {ProductName} version {ProductVersion}
+
+|===
+|Component |Version
+
+|Jaeger
+|1.28.0
+
+|OpenTelemetry
+|0.33.0
```

Comment:
![image](https://user-images.githubusercontent.com/1004403/142361557-be3b62af-778b-44e6-ab82-8b4921422f36.png)
I think component name should be 
* {ProductName} platform
* {ProductName} data collection

CC @pavolloffay @mwringe

Reply:
This table is referring to the upstream version being shipped with the product, so I think that the unbranded names should be used here.